extensions/blocks/pinterest/edit.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { Placeholder, SandBox, Button, IconButton, Toolbar } from '@wordpress/components';
+import { BlockControls } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { pinType } from './utils';
+import { icon } from '.';
+import './editor.scss';
+
+class PinterestEdit extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.hideOverlay = this.hideOverlay.bind( this );
```

Comment:
Do you think it'd be possible to drop binding and switch to using class properties / arrow functions instead?

Reply:
Yap, better have things consistent in Calypso even if core uses bindings. ;-)

Reply:
👍🏻 d68b9381a